hangman.js

```diff
@@ -1 +1,96 @@
-console.log("welcome to hangman");
\ No newline at end of file
+console.log("welcome to hangman");
+var game = {
+  secretLetters: [],
+  indexLetters: [],
+  //guessedLetters: [],
+  matchingLetters: [],
+  guess: [],
+  wrongGuesses: [],
+
+  getSecretWord: function() {
+    //prompt user for secret word
+    secretWord = prompt("Please enter your secret word.");
```

Comment:
I would make sure to include the var before secretWord to make it a local variable :)